engines/router/missionctl/log/resultlog/kafka_test.go

```diff
@@ -165,27 +163,14 @@ func TestKafkaLoggerWrite(t *testing.T) {
 		topic:               "test-topic",
 		producer:            mp,
 	}
-	testKafkaLogEntry := []byte(`{"key": "value"}`)
-	turingResLogEntry := &TuringResultLogEntry{
-		resultLogMessage: turing.TuringResultLogMessage{},
-	}
-
-	// Patch newKafkaLogEntry
-	monkey.Patch(
-		newJSONKafkaLogEntry,
-		func(message proto.Message) ([]byte, error) {
-			// Test that the function is called with the expected arg
-			assert.Equal(t, &turingResLogEntry.resultLogMessage, message)
-			return testKafkaLogEntry, nil
-		},
-	)
-	defer monkey.UnpatchAll()
+	testKafkaLogEntry := []byte(`{"turing_req_id":"123"}`)
+	msg := &turing.TuringResultLogMessage{TuringReqId: "123"}
 
 	// Set up Produce
 	mp.On("Produce", mock.Anything, mock.Anything).Return(nil)
```

Comment:
instead of `mock.Anything` I think you can use the expected `kafka.Message` in the parameter